pom.xml

```diff
@@ -320,6 +320,11 @@
       <artifactId>grizzled-slf4j_${scala.version.prefix}</artifactId>
       <version>1.0.3</version>
     </dependency>
+
+    <!--
+      Only used directly in tests, but adding test-scope blocks transitive inclusion via Spark, resulting in runtime
```

Comment:
AFAIK, the reason we need it in test or non-test is the same - to use Hadoop outputs?